src/os/unix/pa_unix_util.c

```diff
@@ -270,11 +274,16 @@ PaError PaUnixThread_New( PaUnixThread* self, void* (*threadFunc)( void* ), void
 {
     PaError result = paNoError;
     pthread_attr_t attr;
+    pthread_condattr_t cattr;
     int started = 0;
 
     memset( self, 0, sizeof (PaUnixThread) );
     PaUnixMutex_Initialize( &self->mtx );
-    PA_ASSERT_CALL( pthread_cond_init( &self->cond, NULL ), 0 );
+    PA_ASSERT_CALL( pthread_condattr_init( &cattr ), 0 );
+#if defined(CLOCK_MONOTONIC)
```

Comment:
This allows it to compile on MacOS. But it is probably not the best solution.

```suggestion
#if defined(CLOCK_MONOTONIC) && !defined(__APPLE__)
```

Maybe for Mac, if we cannot implement this function properly then we should assert at runtime.

Reply:
There is some code in chromium's os library for this.  On POSIX, they use a monotonic clock + pthread_cond_timedwait, like I've changed this to use.  On MacOS, which doesn't have that ability, they use `pthread_cond_reltimedwait_np()`.  So I could change to do that on macos.  But this code isn't used on macos.  It seems a waste to add code that's not used.